tests/lib/bin/session-tool

```diff
@@ -121,21 +217,21 @@ busctl \
 	/org/freedesktop/systemd1 \
 	 org.freedesktop.systemd1.Manager \
 	StartTransientUnit "ssa(sv)a(sa(sv))" \
-	"$unit_name" fail 4 \
+	"$unit_name" fail 5 \
```

Comment:
what does 4 vs 5 mean here?

Reply:
This is explained in the busctl manual page but the general idea is that this is a way to encode arguments of type `a` - array. This tells busctl how many arguments are in the array it is now constructing.

Reply:
ah right yes, thanks for the explanation